packages/browser/src/extensions/replay/session-recording.ts

```diff
@@ -243,7 +243,10 @@ export class SessionRecording {
      * instead call `posthog.startSessionRecording({linked_flag: true})`
      * */
     public overrideLinkedFlag() {
-        // TODO what if this gets called before lazy loading is done
+        if (!this._lazyLoadedSessionRecording) {
+            logger.warn('overrideLinkedFlag called before recording has finished loading')
+        }
```

Comment:
do we need a warning if we're storing the value and applying after load?

ah... someone could call it directly
so maybe we should store the override here instead of in posthog core
then it doesn't matter who calls it, it's always either stored or immediately acted on from this class